src/commands/support_command.py

```diff
@@ -0,0 +1,34 @@
+from ticket_command import ticket_command
+
+async def support_command(ctx, self, guild):
+    customer = ctx.author
+
+    help_message = f'@{customer.name}, How can I help you? React to this message to receive assistance:\n🔧: Wrench Stuff\n💡: Bulb Stuff'
+    
+    # U0001F527 = Wrench | U0001F4A1 = Bulb
+    emojis = ['\U0001F527','\U0001F4A1']
```

Comment:
I would make this a dictionary so you can reference the emojis like the following; makes your code more readable

emojis = {'wrench': '\U0001F527', 'bulb': '\U0001F4A1'}
emoji['wrench']
emoji['bulb']